boards/silabs/radio_boards/siwx917_rb4338a/Kconfig.defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2023 Antmicro
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_SIWX917_RB4338A
+
+config BOARD
+	default "siwx917_rb4338a"
```

Comment:
this is hwmv1

Reply:
Done. Removed the whole (now empty) file.